src/gui/mainwindow.cpp

```diff
@@ -740,9 +740,13 @@ void MainWindow::tabChanged(int newTab)
         qDebug("Changed tab to transfer list, refreshing the list");
         m_propertiesWidget->loadDynamicData();
         m_searchFilterAction->setVisible(true);
+        m_filterLabel->setVisible(true);
```

Comment:
Wouldn't it be better to hide entire `filterColumnWidget`?

Reply:
I tried it now and it doesn't work unless i have to do something different from this PR.

Reply:
Well, it's really not so obvious due to the fact that it's inside the toolbar.
Please look at #18726.

Reply:
Thanks 👍